kpi/models/import_export_task.py

```diff
@@ -546,6 +546,7 @@ class ExportTaskBase(ImportExportTask):
         # https://github.com/kobotoolbox/kobocat/blob/78133d519f7b7674636c871e3ba5670cd64a7227/onadata/apps/logger/import_tools.py#L67
         '_status',
         '_submitted_by',
+        '__version__',
```

Comment:
This does not add `__version__` to the list of fields in "Select questions to be exported" on the front end. I'll look into doing that

Reply:
https://github.com/kobotoolbox/kpi/pull/4252/commits/8d50eb79385d850a2f75255264afd1939352d170